README.md

```diff
@@ -123,6 +123,11 @@ If your image publishing rate is very slow, you can try
 > export=FASTRTPS_DEFAULT_PROFILES_FILE=<path_to_file>/custom_dds_profile.xml
 > ```
 
+## Optional Automatic Hand Stereo Camera Calibration Routine
+A custom Automatic Hand Stereo Camera Calibration Routine exists in the ```spot_wrapper``` submodule, where the
+output results can be used with ROS 2 for better RGB to Depth correspondence. 
+See the readme at ```/spot_wrapper/spot_spot_wrapper/calibration/README.md``` for more information.
```

Comment:
Not super important but this link won't work on Github  -- can we replace with the URL once the spot_wrapper PR gets merged?